<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ip-172-31-84-83" ignored="2" name="Surefire test" tests="4" failures="1" timestamp="2022-03-19T05:00:08 UTC" time="14.332" errors="0">
  <testcase name="test1" time="0.011" classname="testcases.HomeTest"/>
  <testcase name="ExternalDataTest" time="0.015" classname="testcases.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.LoginTest.ExternalDataTest(LoginTest.java:56)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ExternalDataTest -->
  <testcase name="LoginFailureTest" time="3.502" classname="testcases.LoginTest"/>
  <testcase name="LoginSuccessTest" time="2.783" classname="testcases.LoginTest"/>
  <testcase name="@BeforeTest DataSetUP" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest DataSetUP -->
  <testcase name="@AfterTest DataTearDown" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterTest DataTearDown -->
</testsuite> <!-- Surefire test -->
